.jules/bolt.md

```
@@ -0,0 +1,2 @@
+## Guidelines
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)